src/app/data/cooldowns/cooldown_definitions.ts

```diff
@@ -1,6 +1,6 @@
-export const SUPER_COOLDOWN_RATIO_PER_TIER = [1.44, 1.276, 1.144, 1.0, 0.9488, 0.9024, 0.8608, 0.8224, 0.798, 0.776, 0.7664]
-export const ABILITY_COOLDOWN_RATIO_PER_TIER = [1.25, 1.14, 1.04, 1.0, 0.83, 0.72, 0.62, 0.55, 0.50, 0.455, 0.39]
-export const CLASS_COOLDOWN_RATIO_PER_TIER = [1.427, 1.256, 1.11, 1.0, 0.91461, 0.829, 0.7683, 0.72, 0.622, 0.561, 0.5]
+export const SUPER_COOLDOWN_RATIO_PER_TIER = [1.439, 1.276, 1.144, 1.0, 0.9478, 0.90, 0.8598, 0.8216, 0.798, 0.7751, 0.7651]
+export const ABILITY_COOLDOWN_RATIO_PER_TIER = [1.25, 1.131, 1.039, 1.0, 0.905, 0.831, 0.766, 0.71, 0.642, 0.569, 0.498]
+export const CLASS_COOLDOWN_RATIO_PER_TIER = [1.428, 1.25, 1.111, 1.0, 0.909, 0.8333, 0.7692, 0.7142, 0.666, 0.625, 0.5882]
```

Comment:
![image](https://user-images.githubusercontent.com/88265590/235429722-dec0272e-baba-4cc5-b2c4-0498b999c7d4.png)

According to Clarity, Hunter Dodge does have a different Scalar